app/views/schools/participants/edit_mentor.html.erb

```diff
@@ -0,0 +1,22 @@
+<% content_for :title, "Change mentor" %>
+
+<% content_for :before_content, govuk_back_link(
+  text: "Back",
+  href: schools_cohort_participant_path(@cohort.start_year, @participant))
+%>
+
+<%= form_for @mentor_form, url: schools_cohort_participant_path(@cohort.start_year, @participant), method: :put do |f| %>
+  <%= f.govuk_radio_buttons_fieldset :mentor_id,
+    legend: { text: "Who is the mentor for #{@participant.full_name}?", tag: 'h1', size: 'xl' },
+    hint: { text: "If you don't know, or haven't added them yet you can do this later." } do %>
+    <% @mentors.each do |mentor, i| %>
```

Comment:
Wouldn't `i` be nil here?

Reply:
Heh, is this a js pattern? In our Ruby code? 😂 
Good old javascript passing index as second argument to `forEach` :P

Reply:
at least i didn't write `===` anywhere in this PR!